src/Logger/Logger.cc

```diff
@@ -49,32 +53,78 @@ void InitLogger(bool no_log_file, int verbosity) {
         case 3:
             stdout_logger->set_level(spdlog::level::warn);
             break;
+        case 4:
+            stdout_logger->set_level(spdlog::level::info);
+            break;
         case 5:
             stdout_logger->set_level(spdlog::level::debug);
             break;
-        case 6:
-            stdout_logger->set_level(spdlog::level::trace);
-            break;
         default: {
             stdout_logger->set_level(spdlog::level::info);
             break;
         }
     }
 
```

Comment:
Please also set the level of the performance logger to the same level as the default logger.

At the moment there is an inconsistency between the two flags: if log_protocol_messages is enabled, but the log level is < debug, no protocol messages are printed -- which is the behaviour I would expect, because the protocol messages are logged at the debug level. But because the performance messages are implemented as a separate logger, if log_performance is enabled they are always logged, even if the log level is < info, even though they are now info level messages. We should set the perf logger level to the same level, so that the verbosity flag always takes precedence.